src/Symfony/Bundle/WebServerBundle/README.md

```diff
@@ -1,6 +1,11 @@
 WebServerBundle
 ===============
 
+**CAUTION**: this bundle is deprecated since Symfony 4.4. Instead, use the
+[Symfony Local Web Server](https://symfony.com/doc/current/setup/symfony_server.html).
+
+-----
```

Comment:
This line and the next one should be removed

Reply:
Ok I will do this 👍🏻
FYI I took inspiration at https://github.com/symfony/inflector